taxi_service/settings.py

```diff
@@ -133,3 +133,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+LOGIN_REDIRECT_URL = "/"
```

Comment:
The `LOGIN_REDIRECT_URL` is set to `/`, which means users will be redirected to the home page after logging in. Ensure this is the desired behavior for your application.